content/influxdb/v2.0/reference/release-notes/flux.md

```diff
@@ -8,6 +8,65 @@ menu:
     name: Flux
 ---
 
+## v0.88.0 [2020-10-05]
+
+### Features
+- Moves functions from `v1` package to `schema` package.
+
+### Bug fixes
+- Fix field type error in test.
+- Update buildinfo script to handle new and deleted files.
+- Sets default quantile method when not specified.
+- Improve security of Dockerfile for build scripts.
+
+---
+
+## v0.87.1 [2020-10-01]
+
+### Bug fixes
+- Fetch ragel dependency over HTTPS.
+- Ensure `ast.TextPart` is properly escaped when formatting.
+- Elapsed with multiple buffers per table.
+
+---
+
+## v0.87.0 [2020-09-28]
+
+### Features
+- Linear interpolation.
+- Type signature for linear interpolate function.
+
+### Bug fixes
+- Fix compiler type inference with extended records.
+- Colm Flux grammar updates, keywords, string interpolation, UTF8 IDs.
+- Exponent operator have higher precedence.
+
+---
+
+## v0.86.0 [2020-09-21]
+
+### Features
+- Add operator profiler.
+- Add duration conversion.
+- Add naive bayes classification.
+
+### Bug fixes
+- Reset pointer after scanning invalid Unicode.
+- Catch references to non-existent columns.
+- Propagate span context to `source.Run`.
+
+---
+
+## v0.85.0 [2020-09-14]
+
+### Features
+- Add `Aggregate.window` for an alternative windowing aggregate.
```

Comment:
Add to [aggregateWindow() function](https://docs.influxdata.com/influxdb/v2.0/reference/flux/stdlib/built-in/transformations/aggregates/aggregatewindow/)?